package.json

```diff
@@ -22,25 +22,18 @@
     "Aadit M Shah <aaditmshah@aadit.codes> (https://aadit.codes/)"
   ],
   "scripts": {
-    "format": "prettier --write . && npm run lint -- --fix",
+    "format": "prettier --write . && eslint . --fix",
```

Comment:
Went back to just calling `eslint` as the `lint` was a recursive call to this same task, and also errored out because `--fix` is unrecognized by prettier

Reply:
Thanks for identifying this. I filed #487 in case there are issues with the other aspects.

Reply:
No problem, just ping me if that lands and you want me to rebase